src/contributors/RahmatFathoni.kt

```diff
@@ -0,0 +1,90 @@
+package com.linov.community.contributors
+
+import com.linov.community.Contributor
+import com.linov.community.Educations
+import com.linov.community.Jobs
+import com.linov.community.Skills
+import io.ktor.application.ApplicationCall
+import io.ktor.mustache.MustacheContent
+import io.ktor.response.respond
+
+/**
```

Comment:
ini di remove saja mas...

Reply:
kalo d hapus nanti alamat http://0.0.0.0:8080/hack/rahmatf jadi not found mas? emg bgitu?